### DISCUSSION TITLE
Nest Thermostat

### DISCUSSION THREAD

**Discussion by Ashonts** [balansse/homebridge-vivint#111] [Q&A] [answered]
I have a nest thermostat combined with a house fan, currently everything works but one issue i have is when turning on the fan from home bridge, the fan has a max of 30 min on instead of the 12 hours the thermostat supports

**Comment by balansse** [comment, marked as answer]
Vivint integration with Nest thermostats is not perfect and not feature-rich. I suggest ignoring the thermostat in the plugin settings and using [homebridge-nest](https://github.com/chrisjshull/homebridge-nest) plugin to add the thermostat.
